compiler/damlc/tests/daml-test-files/ClassMethodNames.daml

```diff
@@ -0,0 +1,26 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+--
+-- Ensure that class method names are preserved during the translation
+-- to DAML-LF.
+--
+-- @SINCE-LF 1.7
+-- @QUERY-LF (.interned_strings | map(select(. == "m_baz")) == ["m_baz"])
+-- @QUERY-LF (.interned_strings | map(select(. == "m_bar")) == ["m_bar"])
+-- @QUERY-LF (.interned_strings | map(select(. == "m_foo1")) == ["m_foo1"])
```

Comment:
Maybe also add a query for the superclass?

Reply:
Done!